### DISCUSSION TITLE
Wipe out

### DISCUSSION THREAD

**Discussion by dcarnelutti** [photoprism/photoprism#2460] [Q&A]
Sorry if this is duplicated. I'm looking for an option to delete everything as I was toying around and now I want to upload the pictures. How can I do this?

**Comment by lastzero** [comment]
You can, for example, run this command in a terminal:

```
docker-compose exec photoprism photoprism reset
```

Related Documentation: https://docs.photoprism.app/getting-started/docker-compose/#examples

**Reply by dcarnelutti** [reply]
Sorry, how can I run that command on OMV? I installed the plugin version

**Reply by lastzero** [reply]
Haven't had a chance to test OMV with our app yet :/ Maybe it provides some functionality to reset data? If not, the maintainer might want to add that as we rather don't want to implement a "delete all" button in the UI for the obvious reason...

**Reply by dcarnelutti** [reply]
Got it. It runs on Podman, not in docker. I'll try to find more info on OMV Forums.

**Comment by dcarnelutti** [comment]
I ran this and there results

sudo podman exec photoprism-app photoprism reset
time="2022-06-24T15:15:28Z" level=info msg="PhotoPrism® needs your support!"
time="2022-06-24T15:15:28Z" level=info msg="Visit https://photoprism.app/membership to learn more."
time="2022-06-24T15:15:28Z" level=warning msg="This will delete and recreate your index database after confirmation"
time="2022-06-24T15:15:28Z" level=warning msg="You will be asked next if you also want to remove cache and sidecar files"

time="2022-06-24T15:15:28Z" level=info msg="keeping index database"

time="2022-06-24T15:15:28Z" level=info msg="keeping cache files"

time="2022-06-24T15:15:28Z" level=info msg="keeping *.json sidecar files"

time="2022-06-24T15:15:28Z" level=info msg="keeping *.yml metadata files"

time="2022-06-24T15:15:28Z" level=info msg="keeping *.yml album files"
time="2022-06-24T15:15:28Z" level=info msg="closed database connection"

**Reply by lastzero** [reply]
"You will be asked next if you also want to remove cache and sidecar files."
👉 It seems asking was not possible because the terminal session was not interactive?

In this case you can run the following:

```
sudo podman exec photoprism-app photoprism reset -y
```

With `-y` you signal your agreement to reset the database without further interaction. However, files can not be deleted this way in case you also want to recover all storage that has been used for caching, remove the YAML backups of albums etc. See command output.

**Reply by dcarnelutti** [reply]
Thanks! I ended up deleting the whole plugin and reinstalling it.
